### PR TITLE
Update icons.yml

### PR DESCRIPTION
**Pull request by pbenson322**
Corrected titles for UI Icon Library noted in InVision (comments 1-124) with the exception of comment #112 Logo Google and comment #120 Logo StumbleUpon, which I can't find in the above coding. While looking for content to update, I found other friendly names that needed correction, which aren't part of the UI Icon Library. They are as follows: Logo Instagram, Logo OpenShift®, Logo VMware, Logo Yelp, QR code, VMDK disk, WiFi, WiFi off, Covariate, Hashtag, Script, DNA, Erase 3D, MPEG, MPG2, DOC and PPT.

Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

#### Testing / Reviewing

{{ Add descriptions, steps or a checklist for how reviewers can verify this PR works or not }}

### CODE REVIEW

**Review by joshblack — APPROVED** [review]

**Review by abbeyhrt — APPROVED** [review]

File: packages/icons/icons.yml [hunk 1]
```diff
@@ -433,6 +433,7 @@
     - dev
     - development
   sizes:
+    - 16
```
Comment (joshblack) [inline packages/icons/icons.yml]:
```suggestion
```

File: packages/icons/icons.yml [hunk 2]
```diff
@@ -458,6 +459,7 @@
     - trash
     - edit
   sizes:
+    - 16
```
Comment (joshblack) [inline packages/icons/icons.yml]:
```suggestion
```

File: packages/icons/icons.yml [hunk 3]
```diff
@@ -548,6 +550,7 @@
 - name: edit--off
   friendly_name: Edit off
   sizes:
+    - 16
```
Comment (joshblack) [inline packages/icons/icons.yml]:
```suggestion
```

File: packages/icons/icons.yml [hunk 4]
```diff
@@ -562,6 +565,7 @@
     - post
     - send
   sizes:
+    - 16
```
Comment (joshblack) [inline packages/icons/icons.yml]:
```suggestion
```

File: packages/icons/icons.yml [hunk 5]
```diff
@@ -623,10 +627,12 @@
     - fave
     - bookmark
   sizes:
+    - 16
```
Comment (joshblack) [inline packages/icons/icons.yml]:
```suggestion
```

File: packages/icons/icons.yml [hunk 6]
```diff
@@ -623,10 +627,12 @@
...
     - 32
 - name: favorite--filled
   friendly_name: Favorite filled
   sizes:
+    - 16
```
Comment (joshblack) [inline packages/icons/icons.yml]:
```suggestion
```

File: packages/icons/icons.yml [hunk 7]
```diff
@@ -697,6 +703,7 @@
     - 2x grid
     - pattern
   sizes:
+    - 16
```
Comment (joshblack) [inline packages/icons/icons.yml]:
```suggestion
```

File: packages/icons/icons.yml [hunk 8]
```diff
@@ -870,6 +877,7 @@
     - edit
     - padlock
   sizes:
+    - 16
```
Comment (joshblack) [inline packages/icons/icons.yml]:
```suggestion
```

File: packages/icons/icons.yml [hunk 9]
```diff
@@ -1386,6 +1394,7 @@
     - destroy
     - remove
   sizes:
+    - 16
```
Comment (joshblack) [inline packages/icons/icons.yml]:
```suggestion
```

**Review by joshblack — CHANGES_REQUESTED** [review]

**Review by connor-leech — APPROVED** [review]

**Review by joshblack — APPROVED** [review]

**Review by emyarod — APPROVED** [review]
name changes look good to me

**Pull request merged**